RxASDataSources.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
      s.name = 'RxASDataSources'
-     s.version = '0.3.4'
+     s.version = '0.4.1'
```

Comment:
Could you please revert pod version or help to combine with PR #35 for version 0.4 that supports Swift 5?

Reply:
@dangthaison91 Done